scripts/ci_script.sh

```diff
@@ -12,6 +12,7 @@ fi
 
 
 if [[ $GROUP == python ]]; then
+    jupyter lab build --debug
```

Comment:
Did we want to keep this, or was it just for debugging? If the latter, we might shave off some CI time by removing it again.